packages/gatsby-theme-carbon/src/components/markdown/Markdown.module.scss

```diff
@@ -169,6 +174,14 @@
 :global(.container--dark) .blockquote .list-item {
   color: $carbon--blue-40;
 }
+
+:global(.bx--list--ordered:not(.bx--list--nested) > .bx--list__item::before) {
+  display: none;
+}
+
+:global(.bx--list--ordered:not(.bx--list--nested)) .list-item {
+  margin-left: 0.5rem;
+}
```

Comment:
Could we combine these two selectors, or target them in a way that doesn't require global styles?